src/commands/functions-delete.ts

```diff
@@ -0,0 +1,88 @@
+"use strict";
+
+import * as _ from "lodash";
+
+import { Command } from "../command";
+import * as clc from "cli-color";
+import * as cloudfunctions from "../gcp/cloudfunctions";
+import * as functionsConfig from "../functionsConfig";
+import { deleteFunctions } from "../functionsDelete";
```

Comment:
Good point, its super unclear and there probably isn't a good reason to keep the old names. 

FWIW:
`functions-delete` follows the pattern we use for files defining the different firebase commands - `firebase functions:delete` is defined by the file `functions-delete.ts`, `firebase database:get` is defined by the file `database-get`, etc. 
`functionsDelete` is just a file that contains code to delete functions - my guess is that whoever originally wrote the file named it as such so that it would show up in their IDE next to all of the other functions* files like `functionsConfig`, `functionsDeployHelper`, etc.

Reply:
If I had a time machine I'd put `functions-delete.ts` in `commands/functions/delete.ts` but the ship has sailed.